README.md

```diff
@@ -1,17 +1,10 @@
 # Baby Steps with Rust - UDP Datagram
 
-Hey there! I'm Tim. I primarily work as a Ruby on Rails developer, but I've been excited by Rust for a long time. And I have dabbled. Oh, how I have dabbled. But recently I've felt like it was time to write some more substantial Rust code, so I came up with some babysteps...
-
-Any this is the first babystep! .
-
 One thing that attracts me to Rust is that it is a systems language that you can also comfortably write web apps in. So, I wanted to start with something systems-y, but very small. Something with bytes! This led me to chose a very simple networking protocol: UDP.
 
 UDP is User Datagram Protocol, and was created as part of the Internet Protocol Suite.
 
-Breaking down its name further we learn that:
-- User - highlights that this protocol is for the User or Application layer
```

Comment:
If this wasn't already deleted, it'd be the biggest error in the script that I can see.
TCP and UDP are transport layer protocols, both working on top of IP, which is a network layer protocol:

1. Physical
2. Data-link
3. Network
4. Transport
5. Application

Reply:
Ooo, good catch, and good to know!
I was this many days old when I first needed to know the different levels of the OSI model 😂